src/CsvReader.php

```diff
@@ -253,6 +170,10 @@ protected function openCsvFile($mode)
                 Exception::FILE_NOT_EXISTS_STR
             );
         }
+        $this->lineBreak = $this->detectLineBreak();
+        rewind($this->getFilePointer());
```

Comment:
proč je tady jiný rewind než na L176?

Reply:
tj. proč tady není `$this->rewind()`

Reply:
protoze to dela neco jinyho - header je vzdycky prvni radek, ale $this->rewind respektuje skip lines, kdyby se tady volalo $this->rewind, tak bude header nejaka blbost